pom.xml

```diff
@@ -97,16 +97,16 @@
     <ver.guava>32.1.3-jre</ver.guava>
 
     <ver.gson>2.10.1</ver.gson>
-    <ver.commons-io>2.14.0</ver.commons-io>
-    <ver.commons-cli>1.5.0</ver.commons-cli>
+    <ver.commons-io>2.15.0</ver.commons-io>
+    <ver.commons-cli>1.6.0</ver.commons-cli>
```

Comment:
Are these the same as the Jena5 dependencies?  if not - could you explain what's going on?